api/models/post/updatePost.js

```diff
@@ -1,13 +1,14 @@
 const { GraphQLYogaError } = require('@graphql-yoga/node')
 import setupPostAttrs from './setupPostAttrs'
 import updateChildren from './updateChildren'
+import { isEqual } from 'lodash'
 import {
   updateGroups,
   updateAllMedia,
   updateFollowers
 } from './util'
 
-export default function updatePost (userId, id, params) {
+export default async function updatePost (userId, id, params) {
```

Comment:
Not sure why this async is needed? DIdn't see any awaits here or is it needed because of the tests?

Reply:
Good catch -- I removed the async and fixed the test